programs/account-compression/src/processor/initialize_nullifier_queue.rs

```diff
@@ -59,29 +68,32 @@ pub fn process_initialize_nullifier_queue<'a, 'b, 'c: 'info, 'info>(
     Ok(())
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(AnchorDeserialize, AnchorSerialize, Debug, PartialEq)]
 #[account(zero_copy)]
 #[aligned_sized(anchor)]
 pub struct NullifierQueueAccount {
-    pub index: u64,
-    pub rollover_fee: u64,
-    /// Tip for maintaining the account.
-    pub tip: u64,
-    /// The slot when the account was rolled over, a rolled over account should not be written to.
-    pub rolledover_slot: u64,
-    pub owner: Pubkey,
-    pub delegate: Pubkey,
-    pub associated_merkle_tree: Pubkey,
-    pub next_queue: Pubkey,
+    pub metadata: QueueMetadata,
```

Comment:
could we use the same data structur in the assert function?
(This way we can add standards for these instead of adding an additional struct as I propose in a different comment.)